Android.mk

```diff
@@ -0,0 +1,26 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := IchnaeaNlp
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := IchnaeaNlp
+
+ichnaea_root  := $(LOCAL_PATH)
+ichnaea_out   := $(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
+ichnaea_build := $(ichnaea_root)/build
+ichnaea_apk   := build/outputs/apk/IchnaeaNlp-release-unsigned.apk
```

Comment:
The final apk name, as it ends up in `/system/app` indeed is defined by that variable, but the output of Gradle is not set by that variable but by the name of the directory. As you usually check out a repo with name IchnaeaNlpBackend in a directory called IchnaeaNlpBackend, it should end up with this name.